vaadin-list-mixin.html

```diff
@@ -271,11 +329,18 @@
       this._scroll(scrollDistance);
     }
 
-    /* @protected */
+    /**
+     * @protected
+     * @return {boolean}
```

Comment:
In this PR there seem to be 11 cases where `@protected` is the last annotation, 2 where it's the only annotation and 3 where it is the first annotation. For consistency I'd suggest to always have `@public`/`@protected`/`@private` as either the first or last annotation so for least changes I would make it always the last one.

```suggestion
     * @return {boolean}
     * @protected
```

Not a blocker though.

Reply:
Good catch, somehow I missed that. Updated to make `@protected` always the last one.